packages/as-types/assembly/argument.ts

```diff
@@ -287,6 +308,13 @@ export class Args {
       this.serialized = this.serialized.concat(f32ToBytes(arg as f32));
     } else if (arg instanceof f64) {
       this.serialized = this.serialized.concat(f64ToBytes(arg as f64));
+      // @ts-ignore isDefined is an assemblyscript function
+    } else if (isDefined(arg.serialize)) {
```

Comment:
Is it solved at compilation time or during run execution ?

Reply:
Documentation says: 
> Tests if the specified expression resolves to a defined element. Compiles to a constant.

So I guess it's at compilation time

Reply:
Instanceof is ok in assemblyscrit